### ISSUE TITLE
Respond gracefully when benchmarks not set up on baseline

### ISSUE THREAD

**Issue by epatters** [tkf/BenchmarkCI.jl#89]
Thanks for this useful package! I am currently setting up both PkgBenchmark and BenchmarkCI in the same branch. As a result, the folder `benchmark` does not exist on the master branch. This causes baseline run of benchmarks to fail on pending PR.

I had to dig into the BenchmarkCI source code to figure out what was going wrong. It would be nice if a more informative error, or even just a warning, was raised when the baseline branch is not set up for benchmarks.

**Comment by tkf** [comment]
Yeah, this has been in my (although rather low-priority) TODO #37 but let's track it here.

**Comment by epatters** [comment]
Whoops, didn't see that. Thanks.

**Comment by filchristou** [comment]
I guess we can close this :) ?
